srcts/extras/busy-indicators/busy-indicators.scss

```diff
@@ -46,6 +50,59 @@
   }
 }
 
+@mixin spinner-svg($name) {
+  [data-shiny-busy-spinners="#{$name}"] {
+    --_shiny-spinner-url: url("spinners/#{$name}.svg");
+  }
+}
+
+@include spinner-svg("12-dots-scale-rotate");
+@include spinner-svg("180-ring-with-bg");
+@include spinner-svg("180-ring");
+@include spinner-svg("270-ring-with-bg");
+@include spinner-svg("270-ring");
```

Comment:
I'm not sure about including all the spinners here -- it would increase the CSS size almost 10x. How about we identify a handful that we like and document how to bring in others?

Reply:
I moved these into the `busyIndicatorDependency()` files, which means that we only inline the default spinner.

I agree it's a lot of choices, but I like choice and I don't think there's much marginal benefit to reducing the number of options with this new approach.